### DISCUSSION TITLE
REGARDING SELF LEARNING

### DISCUSSION THREAD

**Discussion by MiruthinaRashmi** [github/india#304] [GitHub Platform] [answered]
heyy guys!!
i have completed the first sub module **Describe core Azure concepts** but when i check in frt portal it is manifesting "IN PROGRESS"
if anyone of u out there knows how to resolve this issue kindly respond
![Screenshot (418)](https://user-images.githubusercontent.com/91934103/136338663-45057458-78f1-4a2a-8e02-1bcfd8cdc407.png)
![Screenshot (419)](https://user-images.githubusercontent.com/91934103/136338671-eece478b-486a-4e37-8323-f486b110891b.png)
![Screenshot (420)](https://user-images.githubusercontent.com/91934103/136338673-134f62c7-4ab8-467a-ba9e-7a08d2523bb9.png)
![Screenshot (421)](https://user-images.githubusercontent.com/91934103/136338677-75961a3f-c5cf-4073-8d22-3d64898c6fe7.png)

**Comment by Soumajit-Roy** [comment, marked as answer]
it will take a week to update on the portal

**Reply by MiruthinaRashmi** [reply]
Ohh kk thanks:)

**Reply by PRIYA-S-102** [reply]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together!
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Reply by MiruthinaRashmi** [reply]
Thnks for the info:)

**Reply by PRIYA-S-102** [reply]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
